registry/metrics.go

```diff
@@ -35,19 +34,19 @@ func NewMetrics() Metrics {
 			Namespace: "flux",
 			Subsystem: "registry",
 			Name:      "fetch_duration_seconds",
-			Help:      "Duration of image metadata fetches, in seconds.",
+			Help:      "Duration of Image metadata fetches, in seconds.",
 			Buckets:   stdprometheus.DefBuckets,
 		}, []string{fluxmetrics.LabelInstanceID, LabelRepository, fluxmetrics.LabelSuccess}),
 		RequestDuration: prometheus.NewHistogramFrom(stdprometheus.HistogramOpts{
 			Namespace: "flux",
 			Subsystem: "registry",
 			Name:      "request_duration_seconds",
-			Help:      "Duration of HTTP requests made in the course of fetching image metadata",
+			Help:      "Duration of HTTP requests made in the course of fetching Image metadata",
 		}, []string{fluxmetrics.LabelInstanceID, LabelRepository, LabelRequestKind, fluxmetrics.LabelSuccess}),
 	}
 }
 
-func (m Metrics) WithInstanceID(instanceID flux.InstanceID) Metrics {
+func (m Metrics) WithInstanceID(instanceID string) Metrics {
```

Comment:
To remove the dependency on the flux God-package. 
I just checked and the new cache re-introduces the dependency again.